internal/snmp/translate.go

```diff
@@ -169,6 +174,10 @@ func SnmpTranslateCall(oid string) (mibName string, oidNum string, oidText strin
 			return oid, oid, oid, oid, out, err
 		}
 
+		if out.RenderNumeric() == "" {
+			return oid, oid, oid, oid, out, fmt.Errorf("cannot make %v numberic, please ensure all imported mibs are in the path", oid)
+		}
+
 		oidNum = "." + out.RenderNumeric() + end
```

Comment:
What do you think of this? Only render the OID once.
```suggestion
		if oidNum = out.RenderNumeric(); oidNum == "" {
			return oid, oid, oid, oid, out, fmt.Errorf("cannot make %v numeric, please ensure all imported mibs are in the path", oid)
		}

		oidNum = "." + oidNum + end
```

Reply:
you got it